src/tests/electron/tests/unified-settings-panel.test.ts

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { createApplication } from 'tests/electron/common/create-application';
+import { scanForAccessibilityIssues } from 'tests/electron/common/scan-for-accessibility-issues';
+import { AppController } from 'tests/electron/common/view-controllers/app-controller';
+import { AutomatedChecksViewController } from 'tests/electron/common/view-controllers/automated-checks-view-controller';
+import { CommonSelectors } from 'tests/end-to-end/common/element-identifiers/common-selectors';
+import { settingsPanelSelectors } from 'tests/end-to-end/common/element-identifiers/details-view-selectors';
+
+describe('AutomatedChecksView -> Settings Panel', () => {
+    let app: AppController;
+    let automatedChecksView: AutomatedChecksViewController;
+
+    beforeEach(async () => {
+        app = await createApplication();
+        automatedChecksView = await app.openAutomatedChecksView();
+        await automatedChecksView.waitForViewVisible();
+        await automatedChecksView.openSettingsPanel();
+    });
+
+    afterEach(async () => {
+        automatedChecksView = null;
+        if (app != null) {
+            await app.stop();
+        }
+    });
+
+    describe('Telemetry toggle', () => {
+        it('should default to "on" in the usual configuration our E2E tests use', async () => {
+            await automatedChecksView.expectToggleState(
+                settingsPanelSelectors.telemetryStateToggle,
+                true,
+            );
+        });
+    });
+
+    describe('High Contrast Mode toggle', () => {
+        it('should default to non-high-contrast mode', async () => {
```

Comment:
Technically, it should default to the system's high contrast mode, but I'm not sure we have a good way of querying that from the test process...

Reply:
If there isn't a good way to have the test do the right thing, it might be nice to at least use a specific error message to explain that case

Reply:
Skipping the test for now